src/portable/portable.c

```diff
@@ -1080,6 +1122,8 @@ static int attach_unit_file(
                 if (r < 0)
                         return log_debug_errno(r, "Failed to install unit file '%s': %m", path);
 
+                mac_selinux_create_file_clear();
```

Comment:
move immediately after the open_tmpfile_linkable() so that we don't forget to call it in error paths